jenkins/build-trigger.jpl

```diff
@@ -245,8 +245,8 @@ node("defconfig-creator") {
 
             addDefconfigs(configs, kdir, arch)
 
-            if (params.TREE != "stable" && params.TREE != "stable-rc")
-                addExtraConfigs(configs, kdir, arch, params.TREE)
```

Comment:
Why not pass params.TREE_NAME to the function instead?  It's better not to use global variables whenever possible.